vulnerabilities/importers/vulnrichment.py

```diff
@@ -0,0 +1,264 @@
+                        system=scoring_system, scoring_elements=vector_string, value=decision
+                    )
+                    severities.append(severity)
+                # ignore kev
+            else:
+                vector_string = metric_value.get("vectorString")
+                base_score = metric_value.get("baseScore")
+                scoring_system = vulnrichment_scoring_system[metric_type]
+                severity = VulnerabilitySeverity(
+                    system=scoring_system, value=base_score, scoring_elements=vector_string
+                )
+                severities.append(severity)
+
+    # Extract references cpes and ignore affected products
+    cpes = set()
+    for affected_product in cna_data.get("affected", []):
+        if type(affected_product) != dict:
+            continue
+        cpes.update(affected_product.get("cpes") or [])  # TODO Add references cpes
+
+    # TODO ADD reference type
+    references = [
+        Reference(url=ref.get("url"), severities=severities)
+        for ref in cna_data.get("references", [])
+    ]
+
+    cpes_ref = [
+        Reference(
+            reference_id=cpe,
+            url=f"https://nvd.nist.gov/vuln/search/results?adv_search=true&isCpeNameSearch=true&query={cpe}",
+        )
+        for cpe in sorted(list(cpes))
+    ]
+    references.extend(cpes_ref)
+
+    weaknesses = set()
+    for problem_type in cna_data.get("problemTypes", []):
+        descriptions = problem_type.get("descriptions", [])
+        for description in descriptions:
+            cwe_id = description.get("cweId")
+            if cwe_id:
+                weaknesses.add(get_cwe_id(cwe_id))
+
+            description_text = description.get("description")
+            if description_text:
+                pattern = r"CWE-(\d+)"
+                match = re.search(pattern, description_text)
+                if match:
+                    weaknesses.add(int(match.group(1)))
+
+    return AdvisoryData(
+        aliases=[cve_id],
+        summary=summary,
+        references=references,
+        date_published=date_published,
+        weaknesses=list(weaknesses),
+        url=advisory_url,
+    )
+
+
+def ssvc_calculator(ssvc_data):
+    """
+    Return the ssvc vector and the decision value
+    """
+    options = ssvc_data.get("options", [])
+    timestamp = ssvc_data.get("timestamp")
+
+    # Extract the options into a dictionary
+    options_dict = {list(option.keys())[0]: list(option.values())[0].lower() for option in options}
+
+    # Determining Mission and Well-Being Impact Value
+    mission_well_being_table = {
+        # (Mission Prevalence, Public Well-being Impact) : "Mission & Well-being"
+        ("minimal", "minimal"): "low",
+        ("minimal", "material"): "medium",
+        ("minimal", "irreversible"): "high",
+        ("support", "minimal"): "medium",
+        ("support", "material"): "medium",
+        ("support", "material"): "high",
```

Comment:
Public well-being impact should be `irreversible` here.
```suggestion
        ("support", "irreversible"): "high",
```

Reply:
@keshav-space Good catch!